frontend/src/scenes/insights/insightVizDataLogic.ts

```diff
@@ -138,12 +135,7 @@ export const insightVizDataLogic = kea<insightVizDataLogicType>([
                 display !== ChartDisplayType.WorldMap &&
                 dateRange?.date_from !== 'all',
         ],
-        supportsPercentStackView: [
-            (s) => [s.querySource, s.display],
-            (q, display) =>
-                isTrendsQuery(q) &&
-                PERCENT_STACK_VIEW_DISPLAY_TYPE.includes(display || ChartDisplayType.ActionsLineGraph),
-        ],
+        supportsPercentStackView: [(s) => [s.querySource, s.display], supportsPercentStackView],
```

Comment:
![image](https://github.com/PostHog/posthog/assets/1855120/4c828821-1d66-41ae-a835-90ec8fde32fe)

@Twixes Is there any reason why these simple passthrough util functions need to be written with args like all the ones in this picture?

Reply:
I don't see one, but then JS is full of surprises